indico/htdocs/js/indico/modules/eventmanagement.js

```diff
@@ -37,6 +37,35 @@
             headerTemplate: '',
             sortList: [[1, 0]]
         });
+
+        $('#persons-list .js-count-label:not(.no-role)').qbubble({
+            show: {
+                event: 'mouseover'
+            },
+            hide: {
+                fixed: true,
+                delay: 500,
```

Comment:
This result is really ugly behavior when you hover another item. You still have to wait 500ms for the previous one to close before the new one opens. Shouldn't let's say 100ms be more than enough to move the cursor from the label into the qtip?